rules/prefer-add-event-listener.js

```diff
@@ -44,7 +44,7 @@ const shouldFixBeforeUnload = (assignedExpression, nodeReturnsSomething) => {
 		return false;
 	}
 
-	return !nodeReturnsSomething.get(assignedExpression);
+	return !Reflect.apply(nodeReturnsSomething.get, nodeReturnsSomething, [assignedExpression]);
```

Comment:
This is not an accidental modification, because if this is not changed, the CI test will not pass.

The test case still passes, nothing change.